### ISSUE TITLE
In-App Messaging Support

### ISSUE THREAD

**Issue by iheffernan** [aws-amplify/amplify-swift#2980]
### Is your feature request related to a problem? Please describe.

AWS Pinpoint supports the concept of In-App Messaging campaigns and it appears that this is available in the Amplify JavaScript Library.

Is there a timeline or roadmap item for making this available in Amplify Swift?

We're trying to make decisions between capability in Amplify v. Firebase or even considering a hybrid approach. Including In-App Messaging support in the Swift library might allow us to completely forego Firebase in favor of Amplify.

Any feedback would be appreciated.

### Describe the solution you'd like

Basically the same feature set currently available in the JavaScript library, but for Swift.

### Describe alternatives you've considered

Firebase Cloud Messaging currently supports In-App Messaging. Admittedly, it is still in Beta, but clearly coming soon.

### Is the feature request related to any of the existing Amplify categories?

Analytics

**Comment by harsh62** [comment]
Pinpoint is being deprecated. Closing the issue.
